src/main/java/com/snowflake/snowpark_java/Session.java

```diff
@@ -65,7 +65,7 @@ public static SessionBuilder builder() {
    * @since 0.8.0
    */
   public DataFrame sql(String query) {
-    return new DataFrame(session.sql(query, JavaUtils.objectArrayToSeq(new Object[0])));
+    return new DataFrame(session.sql(query));
```

Comment:
This change guarantees Java code can invoke `Session.sql` function without parameter. So no additional test is required.